powersimdata/network/tests/test_model.py

```diff
@@ -0,0 +1,76 @@
+import pytest
+
+from powersimdata.network.model import area_to_loadzone
+
+
+def test_area_to_loadzone_argument_type():
+    with pytest.raises(TypeError, match="area must be a str"):
+        area_to_loadzone("europe_tub", 3)
+
+    with pytest.raises(TypeError, match="area_type must be either None or str"):
+        area_to_loadzone("europe_tub", "all", area_type=["interconnect"])
+
+
+def test_area_to_loadzone_argument_value():
+    with pytest.raises(ValueError):
+        area_to_loadzone("usa_tamu", "all", area_type="province")
+
+    with pytest.raises(ValueError, match="Invalid area / area_type combination"):
+        area_to_loadzone("usa_tamu", "California", area_type="loadzone")
+
+    with pytest.raises(ValueError, match="Invalid area / area_type combination"):
+        area_to_loadzone("usa_tamu", "WA", area_type="interconnect")
+
+    with pytest.raises(ValueError, match="Invalid area / area_type combination"):
+        area_to_loadzone("europe_tub", "France", area_type="country_abbr")
+
+
+def test_area_to_loadzone():
```

Comment:
I think we should be able to compare two sets directly by `assert set(A) == set(B)` instead of looking into the length of difference being 0?

Reply:
Indeed

Reply:
Are we testing equality here, or whether one is a subset of the other? Maybe we want to be checking `assert set(A) <= set(B)`

Reply:
We are testing equality. I was not sure `==` would be able to compare sets properly, that is why I took the difference and look at the length

Reply:
Done